evaluation/benchmarks/unix50/input/setup.sh

```diff
@@ -2,9 +2,32 @@
 
 set -e
 
-echo 1 10 11 12 2 3 4 5 6 7 8 9.1 9.2 9.3 9.4 9.5 9.6 9.7 9.8 9.9 | xargs -n 1 |
-  sed 's/$/.txt/' | sed 's;^;http://ndr.md/data/unix50/;' | xargs -n1 wget
+PASH_TOP=${PASH_TOP:-$(git rev-parse --show-toplevel)}
 
+## FIXME: These inputs are already 1G when downloaded
+## FIXME: Also, wget is not silent like curl in the other setup scripts.
+
+inputs=(
+  1 10 11 12 2 3 4 5 6 7 8 9.1 9.2 9.3 9.4 9.5 9.6 9.7 9.8 9.9
+)
+
+if [[ "$1" == "-c" ]]; then
+  for input in ${inputs[@]}
+  do
+    rm -f "${input}.txt"
+  done
+  exit
+fi
+
+for input in ${inputs[@]}
+do
+  if [ ! -f "${input}.txt" ]; then
+    wget "http://ndr.md/data/unix50/${input}.txt"
+    "$PASH_TOP/scripts/append_nl_if_not.sh" "${input}.txt"
+  fi
+done
+
+## FIXME: Calling this script with --full is not idempotent.
```

Comment:
There's a more important problem here, that we don't have a way (yet) to get the original small inputs from a source that's not one of our servers. How can we fix that? (if we had that, then everything's solved).